### ISSUE TITLE
1.2.1 Unable to launch

### ISSUE THREAD

**Issue by inklit** [SquidDev-CC/CC-Tweaks#122]
```
[B#347]  Exception in thread "main" [15:57:44] [main/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:-1]: java.lang.NoClassDefFoundError: org/objectweb/asm/commons/ClassRemapper
[B#347]  [15:57:44] [main/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:-1]: 	at org.squiddev.cctweaks.lua.asm.Tweaks.setup(Tweaks.java:21)
[B#347]  [15:57:44] [main/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:-1]: 	at org.squiddev.cctweaks.core.asm.ASMTransformer.<init>(ASMTransformer.java:38)
[B#347]  [15:57:44] [main/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:-1]: 	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[B#347]  [15:57:44] [main/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:-1]: 	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
[B#347]  [15:57:44] [main/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:-1]: 	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
[B#347]  [15:57:44] [main/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:-1]: 	at java.lang.reflect.Constructor.newInstance(Unknown Source)
[B#347]  [15:57:44] [main/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:-1]: 	at java.lang.Class.newInstance(Unknown Source)
[B#347]  [15:57:44] [main/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:-1]: 	at net.minecraftforge.fml.common.asm.ASMTransformerWrapper$TransformerWrapper.<init>(ASMTransformerWrapper.java:218)
[B#347]  [15:57:44] [main/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:-1]: 	at $wrapper.org.squiddev.cctweaks.core.asm.ASMTransformer.<init>(Unknown Source)
[B#347]  [15:57:44] [main/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:-1]: 	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[B#347]  [15:57:44] [main/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:-1]: 	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
[B#347]  [15:57:44] [main/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:-1]: 	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
[B#347]  [15:57:44] [main/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:-1]: 	at java.lang.reflect.Constructor.newInstance(Unknown Source)
[B#347]  [15:57:44] [main/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:-1]: 	at java.lang.Class.newInstance(Unknown Source)
[B#347]  [15:57:44] [main/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:-1]: 	at net.minecraft.launchwrapper.LaunchClassLoader.registerTransformer(LaunchClassLoader.java:88)
[B#347]  [15:57:44] [main/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:-1]: 	at net.minecraftforge.fml.relauncher.CoreModManager$FMLPluginWrapper.injectIntoClassLoader(CoreModManager.java:124)
[B#347]  [15:57:44] [main/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:-1]: 	at net.minecraft.launchwrapper.Launch.launch(Launch.java:115)
[B#347]  [15:57:44] [main/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:-1]: 	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
[B#347]  [15:57:44] [main/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:-1]: Caused by: java.lang.ClassNotFoundException: org.objectweb.asm.commons.ClassRemapper
[B#347]  [15:57:44] [main/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:-1]: 	at java.net.URLClassLoader.findClass(Unknown Source)
[B#347]  [15:57:44] [main/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:-1]: 	at java.lang.ClassLoader.loadClass(Unknown Source)
[B#347]  [15:57:44] [main/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:-1]: 	at sun.misc.Launcher$AppClassLoader.loadClass(Unknown Source)
[B#347]  [15:57:44] [main/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:-1]: 	at java.lang.ClassLoader.loadClass(Unknown Source)
[B#347]  [15:57:44] [main/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:-1]: 	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:106)
[B#347]  [15:57:44] [main/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:-1]: 	at java.lang.ClassLoader.loadClass(Unknown Source)
[B#347]  [15:57:44] [main/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:-1]: 	at java.lang.ClassLoader.loadClass(Unknown Source)
[B#347]  [15:57:44] [main/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:-1]: 	... 18 more
[B#347] Starting download of http://cdn.technicpack.net/platform2/pack-icons/863735.png?1475210628, with 3 tries remaining
```

**Comment by inklit** [comment]
This happens on both the release on CurseForge as well as one that I compiled myself with git.

**Comment by SquidDev** [comment]
Right, I'm using ASM 5.1 whilst 1.8.9 is stuck on 5.0.3. I'll backport the relevant code.

**Comment by inklit** [comment]
Thanks!

**Comment by SquidDev** [comment]
Fixed in 1.2.2
